### ISSUE TITLE
[Feature Request] 1.2.6-dev: Add option to connect support sections with a pillar.

### ISSUE THREAD

**Issue by lordofhyphens** [alexrj/Slic3r#2619]
One of the pieces I am trying to print with support has a very long, narrow overhang. 

Slic3r's support generation is calculating a few very tiny pillars that can easily get knocked loose from the bed. Is it feasible to add an option to try to connect tiny "islands" (say, less than 10mm x 10mm in area) to each other to help them stick better?

**Comment by alexrj** [comment]
Screenshot?    (I heard of this request several times, just want to see the particular situation)
Isn't the base flange enough to connect them?

**Comment by lordofhyphens** [comment]
![screenshot - 02022015 - 11 41 04 am](https://cloud.githubusercontent.com/assets/31754/6005498/76512868-aad0-11e4-8b04-6fd24cf2ecbc.png)

There's the 3d path preview. Obviously if your bed adherence is awesome, it's not a problem. Some of us don't have awesome bed adherence all the time though. 

It probably doesn't need more than 2-3 layers of connection, but this is best left as a configurable item.

**Comment by lordofhyphens** [comment]
@bubnikv your support code addresses this, doesn't it?

**Comment by bubnikv** [comment]
Not exactly. My support code works just differently. I have some tricks in my sleeve to reduce the support density, but that is not implemented yet.

I have currently disabled the pillars in the prusa3d fork completely. The current pillars are too unreliable.
